e2e/__snapshots__/templates.test.js.snap

```diff
@@ -5748,8 +5767,11 @@ function App() {
         <InstantSearch searchClient={searchClient} indexName=\\"indexName\\">
           <div className=\\"search-panel\\">
             <div className=\\"search-panel__filters\\">
-              <RefinementList attribute=\\"facet1\\" />
-              <RefinementList attribute=\\"facet2\\" />
+              <Configure facets={['*']} maxValuesPerFacet={20} />
+              <ExperimentalDynamicWidgets fallbackWidget={RefinementList}>
```

Comment:
Will need to be updated after the release as non exprimental

Reply:
it's already released as non-experimental, but indeed not yet updated here. Separate PR :)